SPEC.md

```diff
@@ -75,10 +75,12 @@ This set of formats makes it easy to build, host and secure an ACI using technol
 - Image archives MUST have only two top-level pathnames, `manifest` (a regular file) and `rootfs` (a directory). Image archives with additional files outside of `rootfs` are not valid.
 - All files in the image MUST maintain all of their original properties, including timestamps, Unix modes, and extended attributes (xattrs).
 - Image archives MAY be compressed with `gzip`, `bzip2`, or `xz`.
-- Image archives MAY be encrypted with AES symmetric encryption, after (optional) compression/encryption.
+- Image archives MAY be encrypted using PGP symmetric encryption with 256-bit key AES cipher, after optional compression.
```

Comment:
Why do we need to be so prescriptive here? I would rather leave it to the operator - since it's symmetric encryption it's going to be controlled at both ends so we should be less opinionated, imho.